doc/ref_model/chapters/chapter04.md

```diff
@@ -331,7 +331,8 @@ The intent of the following Flavours list is to be comprehensive and yet effecti
 <p align="center"><b>Table 4-13:</b> Predefined Compute Flavours.</p>
 
 **1)** These compute Flavours are intended to be used for transitional purposes and workload vendors are expected to consume smaller Flavours and adopt microservices-based designs for their workloads.<br>
-**2)** In Kubernetes based environments these are the resource requests of the containers in the pods. To get guaranteed resources the resource requests should be set to the same values as the resource limits, to get burstable resources the resource limits should be higher than the resource requests while to get best effort resources none of resource requests of resource limits should be set.
+**2)** In Kubernetes based environments these are the resource requests of the containers in the pods. To get guaranteed resources the resource requests should be set to the same values as the resource limits, to get burstable resources the resource limits should be higher than the resource requests while to get best effort resources none of resource requests of resource limits should be set.<br>
+**3)** The disc is local to the VM/POD, which can be either local or external physical storage
```

Comment:
The sentence is a little confusing with the use of "local" twice. Also, typo - typical to use "disc" in Music while "disk"  in IT. Suggestion:
```suggestion
**3)** The disk is local to the VM/POD, which can be either installed on the local server or and external physical storage
```